xarray/core/pdcompat.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pandas as pd
+import pandas.core.algorithms as algos
+
+
```

Comment:
Can you copy the pandas license directly into this file, too? See `coding/cftimesindex.py` for an example.

Reply:
Done